### ISSUE TITLE
How to disable backprop for certain inputs for loss functions with multiple inputs

### ISSUE THREAD

**Issue by tolga-b** [fchollet/keras#3382]
Hi
I have a loss function that takes 3 inputs

```
def myloss(x1):
    def ml(y_true, x2):
        .....
        return l
    return myloss
```

I use the functional API and it works well. However, I was wondering if its possible to disable backprop to for example x1 and only backprop to x2?

Thank you

**Comment by EderSantana** [comment]
try `x1 = K.stop_gradient(x1)`

**Comment by tolga-b** [comment]
Oh, I didn't realize that functionality was already in the backend. I assume this will set all gradients of this function to 0 for x1. Thank you!

**Comment by EderSantana** [comment]
can I see the full code of how you used it?
